ospf6d/ospf6_interface.h

```diff
@@ -205,6 +205,7 @@ extern int backup_seen(struct thread *);
 extern int neighbor_change(struct thread *);
 
 extern void ospf6_interface_init(void);
+extern void ospf6_interface_clear(struct vty *vty, struct interface *ifp);
```

Comment:
struct vty *vty is not used.

It would be better to remove it. It will help https://github.com/FRRouting/frr/pull/8795

Reply:
Done.